Shield/App.xaml.cs

```diff
@@ -55,6 +56,13 @@ sealed partial class App : Application
         /// </summary>
         public App()
         {
+            WindowsAppInitializer.InitializeAsync()
```

Comment:
It looks like exceptions inside AppInsights are automatically handled.  None of the best practices documentation suggests doing this, and as a test I just added 'throw new Exception("foo");' inside UwpDeviceTelemetryInitializer, and while the exceptions were thrown ("Exception thrown: 'System.Exception' in Shield.exe" showed up in debug output), it did not have any effect on the app from the user's perspective (still running, UI fully functional, etc).

I think this one's fine to leave as is, unless you've got a specific scenario in mind?